### PR TITLE
Add entity classes for config management in Backend

### LINKED ISSUE #1528: Support configuration management on the backend.
### What would you like to be added?

Add configuration management backend services

### Why is this needed?

Enables users to configure plug-ins on the backend.

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Feature.

**What this PR does / why we need it?**

Add entity class for configuration management in Backend.

Fixes #1528 

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024-2024 Huawei Technologies Co., Ltd. All rights reserved.
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java]: Sermant Authors.

File: sermant-backend/src/main/java/io/sermant/backend/handler/config/PluginHandler.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * plugin Handler
+ *
+ * @author zhp
+ * @since 2024-05-16
+ */
+public abstract class PluginHandler {
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/handler/config/PluginHandler.java]: PluginConfigHandler

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * configuration information
+ *
+ * @author zhp
+ * @since 2024-05-16
+ */
+@Getter
+@Setter
+public class ConfigInfo {
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java:30]: add description for each filed

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    /**
+     * Interface call successful
+     */
+    SUCCESS("00", "Success."),
+
+    /**
+     * Unable to establish connection with Configuration Center failed
+     */
+    CONNECT_FAIL("01", "Unable to establish connection with Configuration Center failed."),
+
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java]: leave one between Unable and failed

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ServiceType.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+package io.sermant.backend.entity.config;
+
+/**
+ * Enum for DynamicConfigType
+ *
+ * @author zhp
+ * @since 2024-05-22
+ */
+public enum ServiceType {
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ServiceType.java]: ConfigCenterType

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java (hunk 1 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java]: done

File: sermant-backend/src/main/java/io/sermant/backend/handler/config/PluginHandler.java (hunk 2 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/handler/config/PluginHandler.java]: done

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java (hunk 3 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java:30]: done

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ServiceType.java (hunk 5 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ServiceType.java]: done

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java (hunk 4 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java]: done

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+ */
+@Getter
+public enum ResultCodeType {
+    /**
+     * Interface call successful
+     */
+    SUCCESS("00", "Success."),
+
+    /**
+     * Unable to establish connection with Configuration Center failed
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java]: modify this as well

**Review by daizhenyu — DISMISSED** [review]

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java (hunk 6 above)
Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ResultCodeType.java]: done

File: sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    /**
+     * plugin type, the plugin to which the configuration item belongs
+     */
+    private String pluginType;
+
+    /**
+     * The namespace to which the configuration item belongs, only used by the nacos configuration center
+     */
+    private String nameSpace;
```
Comment (lilai23) [inline sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java]: namespace

Comment (hanbingleixue) [reply sermant-backend/src/main/java/io/sermant/backend/entity/config/ConfigInfo.java]: done

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**